pom.xml

```diff
@@ -111,6 +111,7 @@
         <version.joda-time>2.7</version.joda-time>
         <version.jsoup>1.8.1</version.jsoup>
         <version.net.jcip>1.0</version.net.jcip>
+        <version.org.antlr-34>3.4</version.org.antlr-34>
```

Comment:
@scottmarlow note that Hibernate ORM requires ANTLR 2.7 (and generally that's what other apps use too). We use ANTLR 3.4.
Not using the version name could make it ambiguous, so that's why I highlighted the version.
What would you suggest?